Music Player - Minor Update

Bacon-Fixation opened the pull request:
Update
- it will now check if it's about to stream music to an empty channel when a track starts, if there's only 1 member(the bot) it will leave, save on some bandwidth waste :)

Bugfix
- Fixed the "You're in the wrong channel! Join <#null>" when doing `/play` command after the bot was disconnected from the channel with the Right-Click->Disconnect method in discord, also deletes the last Player embed when the bot is d/c in this manner
- I forgot to delete the last embed when a queue is finished :(


Tested On
Windows 10 (Local DB)
Raspian OS (arm64/Local DB)
Debian 10 (arm64/x64/Local DB)

Much Love
-Bacon

galnir commented:
I've rewritten the queue system so it is now stored on Redis for better performance and maintainability. I'll open the PR very soon so you can take a look at it

Bacon-Fixation commented:
Sweet, I am familiar with Redis

Much Love
-Bacon

Bacon-Fixation commented:
closeing as #724 kinda addresses these issues

[d36757c](https://github.com/galnir/Master-Bot/pull/720/commits/d36757c81d6e3938738408bf8e10fe9deffe9154) is the only thing to remember to add back :D

Much Love
-Bacon
